modules/cnv_vm_storage_volume_types.adoc

```diff
@@ -6,12 +6,12 @@
 `ephemeral`::
 A local copy-on-write (COW) image that uses a network volume as a
 read-only backing store. For this Developer Preview, the backing volume
-must be a `PersistentVolumeClaim`. The ephemeral image is created when
+must be a `persistentVolumeClaim`. The ephemeral image is created when
```

Comment:
This is really pedantic but when referring to the object or kubernetes type (as you are here) use `PersistentVolumeClaim` and when referring to the VM disk types in the kubevirt API use `persistentVolumeClaim` as that is how it is spelled in the kubevirt API.

Reply:
Pedantic comments are always welcome. I *think* I fixed this here (same commit as the titling issue): https://github.com/openshift/openshift-docs/pull/12518/commits/4e71a10ddff35d1d0d230ce8471c81b8992919c4